Docs/source/usage/parameters.rst

```diff
@@ -1384,6 +1390,54 @@ External fields
       and :math:`E_z = 0`, and
       :math:`B_x = \mathrm{strength} \cdot y`, :math:`B_y = -\mathrm{strength} \cdot x`, and :math:`B_z = 0`.
 
+Accelerator Lattice
```

Comment:
@dpgrote @cemitch99  let us double check again if we can keep them as close as possible to ImpactX :)
https://impactx.readthedocs.io/en/latest/usage/parameters.html#lattice-elements

Reply:
Looks good on first view to me. See plasmalens -> ConstF below.